future-apps/payroll/contracts/Payroll.sol

```diff
@@ -601,8 +600,8 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
 
                 // Convert amount (in denomination tokens) to payout token and apply allocation
                 uint256 tokenAmount = _totalAmount.mul(exchangeRate).mul(tokenAllocation);
-                // Divide by 100 for the allocation percentage and by ONE for the exchange rate precision
-                tokenAmount = tokenAmount / (100 * ONE);
+                // Divide by 100 for the allocation percentage and by the exchange rate precision
+                tokenAmount = tokenAmount.div(feed.ratePrecision().mul(100));
```

Comment:
```suggestion
                tokenAmount = tokenAmount.div(100).div(feed.ratePrecision());
```

Might be a bit clearer. We could also remove the safe math, since division does nothing.

I was also wondering if it's worth the PITA to do this in assembly so we can use a static call... I can't see the price feed re-entering into this contract, but it's possible it'd be able to change rates on the spot or do something else funny in itself on this call.

Although, of course, we already know the price feed is pretty much a trusted component at this point 😉.

Reply:
Yea, I'd skip the staticcall for now, we'd have to handle reverts as well